.github/workflows/build.yml

```diff
@@ -4,31 +4,40 @@ on:
     branches:
       - main
       - release/**
+      - jvm-matrix
   pull_request:
     branches:
       - '*'
 
 jobs:
   build:
-    name: Build Job ${{ matrix.os }}
+    name: Build Job ${{ matrix.os }} - ${{ matrix.jvm }}
     runs-on: ${{ matrix.os }}
     strategy:
       # we want that the matrix keeps running, default is to cancel them if it fails.
       fail-fast: false
       matrix:
         # TODO: windows-latest
         os: [ubuntu-latest, macos-latest]
+        # Zulu Community distribution of OpenJDK
+        jvm: [ 8.0.302-zulu, 11.0.12-zulu ]
 
     steps:
       - name: Git checkout
         uses: actions/checkout@v2
 
-      # Zulu Community distribution of OpenJDK
-      - name: set up JDK 1.8
-        uses: actions/setup-java@v2
+      - name: "Download JVM: ${{ matrix.jvm }}"
+        uses: sdkman/sdkman-action@master
+        id: sdkman
         with:
-          distribution: 'adopt'
-          java-version: '8'
+          candidate: java
+          version: ${{ matrix.jvm }}
```

Comment:
GH Action already has all the versions, we don't need to install and download sdkman, `actions/setup-java` is enough

Reply:
With SDKMan we can be more specific with versions and distributions. On the other hand, we don't really need it, so agreed - lets stick with what's offered in `setup-java`.